src/bidiMapper/CommandProcessor.ts

```diff
@@ -91,7 +91,7 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEventsMap> {
     this.#parser = parser;
     this.#logger = logger;
 
-    const networkStorage = new NetworkStorage();
+    const networkStorage = new NetworkStorage(browserCdpClient);
```

Comment:
I think the cdp client should be in the networkprocessor and not the storage. Any reason for giving to the storage?

Reply:
`Processor` in Mapper is closer to `CommandProcessor` so `NetworkProcessor` is an entry point for network module commands.